syntax = "proto3";

package Talk;

service Greeter {
rpc SayHello (HelloRequest) returns (HelloReply) {}
}

// The request message containing the user's name.
message HelloRequest {
string name = 1;
string date = 2;
}

// The response message containing the greetings
message HelloReply {
string message = 1;
}